portable/GCC/ARM7_AT91SAM7S/lib_AT91SAM7X256.h

```diff
@@ -1,1469 +1,1396 @@
+/** \fn    AT91F_CKGR_DisableMainOscillator */
+/** \brief Disable the main oscillator */
+/**---------------------------------------------------------------------------- */
+    __inline void AT91F_CKGR_DisableMainOscillator( AT91PS_CKGR pCKGR ) /* \arg pointer to CKGR controller */
+    {
+        pCKGR->CKGR_MOR &= ~AT91C_CKGR_MOSCEN;
+    }
+
+/**---------------------------------------------------------------------------- */
+/** \fn    AT91F_CKGR_CfgMainOscStartUpTime */
+/** \brief Cfg MOR Register according to the main osc startup time */
+/**---------------------------------------------------------------------------- */
+    __inline void AT91F_CKGR_CfgMainOscStartUpTime( AT91PS_CKGR pCKGR,         /* \arg pointer to CKGR controller */
+                                                    unsigned int startup_time, /* \arg main osc startup time in microsecond (us) */
+                                                    unsigned int slowClock )   /* \arg slowClock in Hz */
+    {
+        pCKGR->CKGR_MOR &= ~AT91C_CKGR_OSCOUNT;
+        pCKGR->CKGR_MOR |= ( ( slowClock * startup_time ) / ( 8 * 1000000 ) ) << 8;
+    }
+
+/**---------------------------------------------------------------------------- */
+/** \fn    AT91F_CKGR_GetMainClockFreqReg */
+/** \brief Cfg the main oscillator */
+/**---------------------------------------------------------------------------- */
+    __inline unsigned int AT91F_CKGR_GetMainClockFreqReg( AT91PS_CKGR pCKGR ) /* \arg pointer to CKGR controller */
+    {
+        return pCKGR->CKGR_MCFR;
+    }
+
+/**---------------------------------------------------------------------------- */
+/** \fn    AT91F_CKGR_GetMainClock */
+/** \brief Return Main clock in Hz */
+/**---------------------------------------------------------------------------- */
+    __inline unsigned int AT91F_CKGR_GetMainClock( AT91PS_CKGR pCKGR,       /* \arg pointer to CKGR controller */
+                                                   unsigned int slowClock ) /* \arg slowClock in Hz */
+    {
+        return ( ( pCKGR->CKGR_MCFR & AT91C_CKGR_MAINF ) * slowClock ) >> 4;
+    }
+
+/**---------------------------------------------------------------------------- */
+/** \fn    AT91F_PMC_CfgMCKReg */
+/** \brief Cfg Master Clock Register */
+/**---------------------------------------------------------------------------- */
+    __inline void AT91F_PMC_CfgMCKReg( AT91PS_PMC pPMC, /* \arg pointer to PMC controller */
+                                       unsigned int mode )
+    {
+        pPMC->PMC_MCKR = mode;
+    }
+
+/**---------------------------------------------------------------------------- */
+/** \fn    AT91F_PMC_GetMCKReg */
+/** \brief Return Master Clock Register */
+/**---------------------------------------------------------------------------- */
+    __inline unsigned int AT91F_PMC_GetMCKReg( AT91PS_PMC pPMC ) /* \arg pointer to PMC controller */
```

Comment:
This is formatting makes more sense to me

Reply:
Agree with you, relevant comparison can be found [here](https://github.com/FreeRTOS/FreeRTOS-Kernel/pull/768#discussion_r1313300520)